Dockerfile

```diff
@@ -75,6 +75,18 @@ RUN zypper in -y \
     vim \
     which
 
+# Harvester needs these packages
+RUN zypper in -y apparmor-parser \
```

Comment:
Why don't we put this in the Dockerfile for harvester-os?

Reply:
Both works, putting it here saves some build time in harvester-installer.